components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/FirefoxAccount.kt

```diff
@@ -317,11 +317,13 @@ class FirefoxAccount(handle: FxaHandle, persistCallback: PersistCallback?) : Aut
      * @param sessionToken 64 character string of hex-encoded bytes
      * @param kSync 128 character string of hex-encoded bytes
      * @param kXCS 32 character string of hex-encoded bytes
+     * @return [Boolean] True if the migration network requests succeeded
+     *
      * This performs network requests, and should not be used on the main thread.
      */
-    fun migrateFromSessionToken(sessionToken: String, kSync: String, kXCS: String) {
+    fun migrateFromSessionToken(sessionToken: String, kSync: String, kXCS: String): Boolean {
```

Comment:
I'm not opposed to adding it, but I'm also not sure what the caller is supposed to do with the information in the return value here. Are they supposed to change behaviour based on whether it's true or false, or is it for including in migration-related metrics? If the later than a json blob of metrics data may be a little clearer for consumers, by analogy with the other migration-related components.